### ISSUE TITLE
Remove/rewrite old comments

### ISSUE THREAD

**Issue by SovaPluto** [ookami-kb/gh-pr-size-watcher#13]
Right now on each commit the bot writes a warning/error message. 
Suggested behavior:
If the check runs 2nd/3rd/N times for the same PR it removes previous comments(errors/warnings) and always keeps only the last updated message in history.

**Comment by ookami-kb** [comment]
Hey, yep, that sounds like a good idea to me. I will check it later, or if you have time, feel free to submit a PR.

**Comment by fabian-hering-fnt** [comment]
Any news on this one?

**Comment by ookami-kb** [comment]
Not really, feel free to submit a PR. Otherwise, I can (probably) check it later.
